app/views/spotlight/translations/_general.html.erb

```diff
@@ -66,6 +66,88 @@
         </div>
       <% end %>
     </div>
+    <div class='translation-main-menu'>
+      <h2 class='translation-subheading'>
+        <%= t('spotlight.exhibits.translations.general.main_menu.label') %>
+      </h2>
+
+      <% translation = Translation.find_or_initialize_by(exhibit: current_exhibit, key: "spotlight.curation.nav.home", locale: @language) %>
+      <%= f.fields_for :translations, translation do |translation_fields| %>
+        <%= translation_fields.hidden_field :key %>
+        <%= translation_fields.hidden_field :locale %>
+        <div class='form-group translation-form translation-main-menu-home'>
+          <%= translation_fields.label :value, t('spotlight.exhibits.translations.general.basic_settings.home'), class: 'control-label col-sm-2' %>
+          <div class='col-md-8 panel panel-body panel-translation'>
+            <%= translation_fields.text_field_without_bootstrap :value, class: 'form-control' %>
+            <p class="help-block">
+              <%= t(:'spotlight.curation.nav.home') %>
+            </p>
+          </div>
+          <div class='col-md-2'>
+            <% if translation.value.present? %>
+              <span class='glyphicon glyphicon-ok'></span>
+            <% end %>
+          </div>
+        </div>
+      <% end %>
+      <% translation = Translation.find_or_initialize_by(exhibit: current_exhibit, key: "main_navigation.browse.label", locale: @language) %>
+      <%= f.fields_for :translations, translation do |translation_fields| %>
+        <%= translation_fields.hidden_field :key %>
+        <%= translation_fields.hidden_field :locale %>
+        <div class='form-group translation-form translation-main-menu-browse'>
+          <%= translation_fields.label :value, t('spotlight.exhibits.translations.main_menu.browse'), class: 'control-label col-sm-2' %>
+          <div class='col-md-8 panel panel-body panel-translation'>
+            <%= translation_fields.text_field_without_bootstrap :value, class: 'form-control' %>
+            <p class="help-block">
+              <%= current_exhibit.main_navigations.browse.label %>
```

Comment:
Yeah I was thinking about this a bit. I wonder if we should just consider that the admin pages should only be rendered in the default locale instead of providing for a way for it to also be translated.

Reply:
I worry that could get unnecessarily complicated.

That's one of the reasons I added the Fallbacks module, so that in the absence of an i18n'd value you'll get the English version (which is also the way the behavior has been described in multiple places around the app by @ggeisler).

We've used the `[]` accessor in other parts of the General and Browse categories forms to make this work appropriately under this pattern.

Reply:
Ah that makes total sense. For some reason I had not realized that a multilingual admin UI was expected, but it makes sense why we would want that especially with multiple curators/translators.